### DISCUSSION TITLE
Changing 'Member of'

### DISCUSSION THREAD

**Discussion by StephenFeddes** [YetiForceCompany/YetiForceCRM#17173] [How to]
When I created a contact, I made them a member of an account. However, when I try to change that contact's 'member of' field, it won't let me edit it. What do I do? I have looked inside 'Permissions' and 'Standard Modules'.

**Comment by rskrzypczak** [comment]
Just click one of the buttons highlighted below
![image](https://github.com/YetiForceCompany/YetiForceCRM/assets/10221999/bac6a040-c03d-43e7-b825-5b7a70803a36)
